build/configs/scanners.yaml

```diff
@@ -540,6 +539,13 @@ scanners:
           - 'soap_file'
           - 'html_file' # Simple XML, such as a POM file, is recognized as HTML
       priority: 5
+  'ScanYara':
+    - positive:
+        flavors:
+          - '*'
+      priority: 5
+      options:
+        location: '/etc/yara/'
```

Comment:
Just want to make sure we don't actually need this file, as I don't know of anything that would make it available in any of our deployment targets.

Reply:
```
$ cat /etc/yara/rules.yara
rule test
// This rule verifies the scanYara scanner works. Add rules here for quickstart scanning.
{
  condition:
    true
}$ 
```

It's currently expecting something there... we can change that though.

Reply:
Made it optional!